install-from-binstall-release.sh

```diff
@@ -2,6 +2,37 @@
 
 set -euxo pipefail
 
+if [ "$OS" = "Windows_NT" ]; then
+    # https://github.com/cargo-bins/cargo-binstall/blob/main/install-from-binstall-release.ps1
```

Comment:
This will fix the shellcheck failure:

```suggestion
    # https://github.com/cargo-bins/cargo-binstall/blob/main/install-from-binstall-release.ps1
    # shellcheck disable=SC2016
```